extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -332,14 +332,15 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.config_['vars']['requestCount']++;
 
     // Replace placeholders with URI encoded values.
-    // Precedence is trigger.vars > config.vars.
+    // Precedence is trigger.vars > config.vars > event.vars.
     // Nested expansion not supported.
     request = expandTemplate(request, key => {
       const match = key.match(/([^(]*)(\([^)]*\))?/);
       const name = match[1];
       const argList = match[2] || '';
       const raw = (trigger['vars'] && trigger['vars'][name] ||
-          this.config_['vars'] && this.config_['vars'][name]);
+          this.config_['vars'] && this.config_['vars'][name]) ||
+          event.vars[name];
```

Comment:
Can you describe where `trigger` and `config_['vars']` come from? I'm curious why they're prioritized over the actual event's variables.